_posts/deployment/2017-08-04-deploy-from-archive.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Deploy Directly from an Archive
+modified_at: 2017-08-04 00:00:00
+category: deployment
+tags: source deploy
+order: 1
+---
+
+A way to deploy your application on Scalingo is to use an url where your application code is upload on the format archive.
+
+## The Workflow
+
+If you want to deploy an application without git or github, you can put your application code in an archive `tar.gz`.
+
+Then you need to upload your application on a source, a repo on the web where your application will be conserved. If you do not know how do that, see the section [Create a Repo for the Archive]({% post_url deployment/2017-08-04-deploy-from-archive %}create-a-repo-for-the-archive).
+
+From your source, your repo, you must have a download_url that Scalingo can use to deploy your application.
+
+Thus, you have to provide your download_url on the Scalingo Api (https://api.scalingo.com/v1/apps/[:app]/deployments) with a request POST, see the section [Deployment]({% post_url deployment/2017-08-04-deploy-from-archive %}#to-deploy-an-archive).
```

Comment:
"Thus" means "Ainsi". Not sure that's what you meant here.
"v1/apps/[:app]/deployments": Here you use "[:app]" to designate the user application name. In the next example, you use "[example-app]". Please use something consistent for all this page.
"with a request POST" with a POST request